sstables/sstables.cc

```diff
@@ -1698,7 +1698,7 @@ sstable::read_scylla_metadata(const io_priority_class& pc) noexcept {
 
 void
 sstable::write_scylla_metadata(const io_priority_class& pc, shard_id shard, sstable_enabled_features features, struct run_identifier identifier,
-        std::optional<scylla_metadata::large_data_stats> ld_stats, sstring origin) {
+        std::optional<scylla_metadata::large_data_stats> ld_stats, clustering_position_metadata cpm, sstring origin) {
```

Comment:
why not last parameter?

Reply:
I used optional first, then for symmetry, put optional ld_stats and cpm together. Will move it as last param, no problem.